app/authorization/lib/AuthorizationUtils.ts

```diff
@@ -0,0 +1,48 @@
+import { Meteor } from 'meteor/meteor';
+
+const restrictedRolePermissions = {};
+
+export const AuthorizationUtils = class {
+	static addRolePermissionWhiteList(roleId: string, list: [string]) {
+		if (!roleId) {
+			throw new Meteor.Error('invalid-param');
+		}
+
+		if (!list) {
+			throw new Meteor.Error('invalid-param');
+		}
+
+		if (!restrictedRolePermissions[roleId]) {
+			restrictedRolePermissions[roleId] = { whitelist: [] };
```

Comment:
any reason to have an object here and store the permission on the `whitelist` property instead of storing just an array? I'd also suggest to a `Set` instead of a plain array =)